membership-attribute-service/app/controllers/AccountController.scala

```diff
@@ -222,27 +227,30 @@ class AccountController(
     } yield fromZuora ++ fromStripe
   }
 
-  def fetchCancelledSubscriptions(): Action[AnyContent] =
+  def cancelledSubscriptions(): Action[AnyContent] =
```

Comment:
I renamed this (and a couple of other methods) because there was a method further down the file that just called this method. (I didn't move the code to the other method as the diff is already big)